src/components/search/Search.component.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { clsx } from 'clsx';
+
+import search from '@/assets/icons/search.svg';
+
+import styles from './search.module.css';
+
+const Search = ({ theme }: { theme: string }) => {
+    const nameSelect: string[] = ['Price (High - Low)', 'Price (Low- High)', 'Newest', 'Oldest'];
+
+    const [selectedSort, setSelectedSort] = React.useState<string>(nameSelect[0]);
+    const [isOpenDropDropDown, setIsOpenDropDropDown] = React.useState<boolean>(false);
+
+    // , setSelectedSort] = React.useState<string>(nameSelect[0]);
+    return (
+        <div
+            className={clsx({
+                [styles.main]: true,
+                light_theme: theme === 'light',
+                dark_theme: theme === 'dark',
+            })}
+        >
+            <div className={styles.search}>
```

Comment:
<img width="500" alt="Screenshot 2024-05-23 at 13 23 38" src="https://github.com/lesiaturuta/fe-react-2024/assets/115080771/a78ef76d-0ca6-41e1-87fa-e776455cfbfd">

the search bar in the mobile version does not match the design

Reply:
fixed